tools/toolchain.sh

```diff
@@ -388,22 +325,48 @@ build_target() {
 	cd "${GCCDIR}"
 	check_error $? "Change directory failed."
 
+	if $USE_HELENOS_TARGET ; then
+		SYSROOT=--with-sysroot --with-build-sysroot="${WORKDIR}/sysroot"
+	else
+		SYSROOT=--without-headers
+	fi
+
 	change_title "GCC: configure (${PLATFORM})"
 	PATH="$PATH:${INSTALL_DIR}/${PREFIX}/bin" "${BASEDIR}/downloads/gcc-${GCC_VERSION}/configure" \
 		"--target=${TARGET}" \
-		"--prefix=${PREFIX}" "--program-prefix=${TARGET}-" \
-		--with-gnu-as --with-gnu-ld --disable-nls --disable-threads \
-		--enable-languages=c,objc,c++,obj-c++ \
-		--disable-multilib --disable-libgcj --without-headers \
-		--disable-shared --enable-lto --disable-werror
+		"--prefix=${PREFIX}" \
+		"--program-prefix=${TARGET}-" \
+		--with-gnu-as \
+		--with-gnu-ld \
+		--disable-nls \
+		--enable-languages=c,objc,c++,obj-c++,go \
```

Comment:
Not sure there is still some interest in objc and obj-c++

Reply:
Good point. I removed them.